README.md

```diff
@@ -1,2 +1,28 @@
-# news-for-maintainers
-Announcements of changes impacting collection contributors and maintainers
+# News for Maintainers
+Announcements of changes impacting Ansible collection contributors and maintainers.
+
+[Current open issues](https://github.com/ansible-collections/news-for-maintainers/issues) and [related discussions](https://github.com/ansible-collections/news-for-maintainers/discussions).
+
+This repository is created in accordance with the [Community decision](https://github.com/ansible-community/community-topics/issues/51) as a replacement for [Issue 45](https://github.com/ansible-collections/overview/issues/45) to track changes that Ansible collection maintainers should be aware of.
+
+If you are a collection maintainer, please subscribe to this repository using the `Watch` button in the upper right corner of the GitHub web interface. If you do not want to get notifications about related discussions, subscribe to issues only.
+
+The news are made via issues - one per announcement.
+
+# Examples of news posted via this repository
+
+- Updates made to ``ansible-test`` to detect new issues.
+- New OS distros available for testing.
+- Changes in test containers.
+- Calls for CI test matrix updates to prevent failing tests.
+- Changes/reminders about freeze and version bump dates.
+- News and required actions related to libraries' deprecation.
+- Changes to `ansibullbot` workflow.
+- Pull request review days and contributor summit announcements.
+- Changes in the changelog format.
+- Collection documentation markup changes.
+
+# Guidelines
+
+- Please do **not** create issues to ask questions. Join us on [Matrix or IRC](https://docs.ansible.com/ansible/latest/community/communication.html#real-time-chat) or create a `Discussion` and link it to the issue instead.
+- Please avoid unnecessary discussions in issues, use the `Discussions` feature as described above. Every comment posted will notify **a lot** of folks! You can add reactions to an issue/a comment instead for feedback that does not annoy everyone.
```

Comment:
Since that regularly happened with the old issue:
```suggestion
- Please avoid unnecessary discussions in issues, use the `Discussions` feature as described above. Every comment posted will notify **a lot** of folks! You can add reactions to an issue/a comment instead for feedback that does not annoy everyone.
- If you reference an issue in this repository in a PR, avoid writing `Fixes` or `Closes` before it so that the issue is not closed when the PR is merged.
```

Reply:
Good catch, committed